### PR TITLE
Added fix for \t in: Fix Unneeded Spaces

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Test/FixCommonErrorsTest.cs [hunk 1]
```diff
@@ -736,16 +736,21 @@ public void FixUnneededSpaces1()
...
 
+        [TestMethod]
+        [DeploymentItem("SubtitleEdit.exe")]
         public void FixUnneededSpaces2()
         {
             using (var target = GetFixCommonErrorsLib())
             {
                 InitializeFixCommonErrorsLine(target, " To be, or not to be!");
                 target.FixUnneededSpaces();
-                Assert.AreEqual(target.Subtitle.Paragraphs[0].Text, " To be, or not to be!");
```
Comment (ivandrofly) [inline src/Test/FixCommonErrorsTest.cs]: Expected shouldn't be ```" To be, or not to be!"```

File: src/Test/FixCommonErrorsTest.cs [hunk 2]
```diff
@@ -779,6 +787,19 @@ public void FixUneededSpaces5()
...
         }
 
+        [TestMethod]
+        [DeploymentItem("SubtitleEdit.exe")]
+        public void FixUneededSpaces6()
+        {
+            using (var target = GetFixCommonErrorsLib())
+            {
+                const string expected = "Foo bar.";
+                InitializeFixCommonErrorsLine(target, "Foo \t\tbar.");
```
Comment (ivandrofly) [inline src/Test/FixCommonErrorsTest.cs]: Fix ```\t```

File: src/Test/FixCommonErrorsTest.cs [hunk 3]
```diff
@@ -736,16 +736,21 @@ public void FixUnneededSpaces1()
...
                 InitializeFixCommonErrorsLine(target, " To be, or not to be!");
                 target.FixUnneededSpaces();
-                Assert.AreEqual(target.Subtitle.Paragraphs[0].Text, " To be, or not to be!");
+                var expected = "To be, or not to be!";
+                Assert.AreEqual(expected, target.Subtitle.Paragraphs[0].Text);
             }
         }
 
+        [TestMethod]
+        [DeploymentItem("SubtitleEdit.exe")]
```
Comment (ivandrofly) [inline src/Test/FixCommonErrorsTest.cs]: Wasn't working

File: src/Test/FixCommonErrorsTest.cs [hunk 4]
```diff
@@ -756,6 +761,8 @@ public void FixUnneededSpaces3()
             }
         }
 
+        [TestMethod]
+        [DeploymentItem("SubtitleEdit.exe")]
```
Comment (niksedk) [inline src/Test/FixCommonErrorsTest.cs]: Why have you added the "DeploymentItem" attr?

Comment (ivandrofly) [inline src/Test/FixCommonErrorsTest.cs]: Since most of test method are using it I thought it was required

Comment (ivandrofly) [inline src/Test/FixCommonErrorsTest.cs]: Deployment attributes removed should I push it on top of this commit?! // @niksedk

**Pull request merged**